webpack/scenes/ContentViews/Details/Repositories/ContentViewRepositories.js

```diff
@@ -60,6 +76,62 @@ const ContentViewRepositories = ({ cvId }) => {
   ];
   const loading = status === STATUS.PENDING;
 
+  const toggleBulkAction = () => {
+    setBulkActionOpen(!bulkActionOpen);
+  };
+  const onAdd = (repos) => {
+    const { repository_ids: repositoryIds } = details;
+    dispatch(updateContentView(cvId, { repository_ids: repositoryIds.concat(repos) }));
+  };
+
+  const onRemove = (repos) => {
+    const reposToDelete = [].concat(repos);
+    const { repository_ids: repositoryIds } = details;
+    const deletedRepos = repositoryIds.filter(x => !reposToDelete.includes(x));
+    dispatch(updateContentView(cvId, { repository_ids: deletedRepos }));
+  };
+
+  const addBulk = () => {
+    const reposToAdd = [];
+    rows.forEach((row) => {
+      if (row.selected) {
+        reposToAdd.push(row.repoId);
+      }
+    });
```

Comment:
I don't mind it spread out, but can be in one line like this `rows.forEach(row => row.selected && reposToAdd.push(row.repoId))`, same can be done in `removeBulk` below

Reply:
actually you could do this:

```javascript
 const addBulk = () => {
    const reposToAdd = rows.filter(row => row.selected);
    onAdd(reposToAdd);
 }

Reply:
its still valid, but the second comment with `rows.filter` avoids initializing the empty array

Reply:
@sjha4 any reason to not use filter here? ^

Reply:
We actually need the repoId on the row in the reposToAdd array. Is there a JS way to pluck that from filtered rows?